cluster/node-pools/worker-karpenter/provisioners.yaml

```diff
@@ -84,7 +84,7 @@ spec:
   instanceProfile: {{ .Cluster.ID | awsValidID }}-WorkerKarpenter-InstanceProfile
   amiFamily: Custom
   amiSelector:
-    Name: {{ .Cluster.ConfigItems.kuberuntu_image_karpenter_label  }}
```

Comment:
@linki 
Can you also remove the kuberuntu_image_karpenter_label from the config_defaults.yaml please